packages/kit/src/components/AccountSelector/NetworkSelectorTrigger/NetworkSelectorTriggerApproval.tsx

```
@@ -0,0 +1,125 @@
+  }, [
+    disabled,
+    openChainSelector,
+    title,
+    selectorNetworks,
+    current?.id,
+    onChange,
+  ]);
+
+  return (
+    <XStack
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)